kili/mutations/label/helpers.py

```diff
@@ -33,3 +35,41 @@ def generate_create_predictions_arguments(
         "model_name_array": [model_name] * len(external_id_array),
         "external_id_array": external_id_array,
     }
+
+
+def get_label_file_paths_to_upload(files: Tuple[str, ...], verbose: bool) -> List[str]:
```

Comment:
All of this is a duplicate of `get_file_paths_to_upload`. Factorise the code, otherwise, when modifying a function, there is a chance that we forget to change the second one.